frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/services/AccountInterventionsService.java

```diff
@@ -0,0 +1,70 @@
+package uk.gov.di.authentication.frontendapi.services;
+
+import com.nimbusds.oauth2.sdk.ParseException;
+import com.nimbusds.oauth2.sdk.http.HTTPRequest;
+import com.nimbusds.oauth2.sdk.http.HTTPResponse;
+import net.minidev.json.JSONObject;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import uk.gov.di.authentication.frontendapi.entity.AccountInterventionsInboundResponse;
+import uk.gov.di.authentication.frontendapi.entity.Intervention;
+import uk.gov.di.authentication.frontendapi.entity.State;
+import uk.gov.di.authentication.shared.exceptions.UnsuccessfulAccountInterventionsResponseException;
+
+import java.io.IOException;
+
+import static java.lang.String.format;
+
+public class AccountInterventionsService {
+
+    private static final Logger LOG = LogManager.getLogger(AccountInterventionsService.class);
+
+    public AccountInterventionsInboundResponse sendAccountInterventionsOutboundRequest(
+            HTTPRequest request) throws UnsuccessfulAccountInterventionsResponseException {
+
+        try {
+            LOG.info("Sending account interventions outbound request");
+            var response = request.send();
+            if (!response.indicatesSuccess()) {
+                throw new UnsuccessfulAccountInterventionsResponseException(
+                        format(
+                                "Error %s when attempting to call Account Interventions outbound endpoint: %s",
+                                response.getStatusCode(), response.getContent()),
+                        response.getStatusCode());
+            }
+            LOG.info("Received successful account interventions outbound response");
+            return parseResponse(response);
+        } catch (IOException e) {
+            throw new UnsuccessfulAccountInterventionsResponseException(
+                    "Error when attempting to call Account Interventions outbound endpoint", e);
+        } catch (ParseException e) {
+            throw new UnsuccessfulAccountInterventionsResponseException(
+                    "Error parsing HTTP response", e);
+        }
+    }
+
+    private AccountInterventionsInboundResponse parseResponse(HTTPResponse response)
+            throws ParseException {
+        var contentAsString = response.getContentAsJSONObject();
+
+        var interventionJson = (JSONObject) contentAsString.get("intervention");
```

Comment:
Might be related to why the above test is passing, or might not, but this get can return a null (as can the `getAsString`s below, and we're not handling this anywhere (assuming maybe it should throw an error?). We should probably write a test case/ some test cases for the case where the interventions json does not look as expected